service/upload/upload.go

```diff
@@ -1,12 +1,15 @@
 package uploadService
 
 import (
+	"regexp"
 	"strings"
 
 	"github.com/NyaaPantsu/nyaa/config"
 	"github.com/NyaaPantsu/nyaa/model"
 )
 
+const trackerRegex = `^(http[s]*|udp)://[a-z0-9\.:\-/?]+$`
```

Comment:
Ok thanks :/. In magnet urls you encode the url of the tracker then? Because the "&" character would not work if not encoded?